examples/tina-cloud-starter/components/tina-wrapper.tsx

```diff
@@ -76,4 +73,82 @@ const Inner = (props) => {
   )
 }
 
+import { Cloudinary } from 'cloudinary-core'
+import {
+  Media,
+  MediaList,
+  MediaListOptions,
+  MediaStore,
+  MediaUploadOptions,
+} from 'tinacms'
+
+export class CloudinaryMediaStore2 implements MediaStore {
+  accept = '*'
+  private api: Cloudinary
+
+  constructor(public cloud_name: string) {
+    this.api = new Cloudinary({
+      cloud_name: this.cloud_name,
+      secure: true,
+    })
+  }
+
+  async persist(media: MediaUploadOptions[]): Promise<Media[]> {
+    const url = `https://api.cloudinary.com/v1_1/${process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME}/image/upload`
```

Comment:
Should this be:
```suggestion
    const url = `https://api.cloudinary.com/v1_1/${this.cloud_name}/image/upload`
```

Reply:
So, actually this declaration isn't actually getting used anywhere. Must have missed pruning it, and I think that was the only instance of using a hard coded URL.